ui/src/main/Footer/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import versionService from "../../services/VersionService/VersionService";
+import Spinner from "react-bootstrap/Spinner";
+import { usePromise } from "react-use-promise-matcher";
+import { BsExclamationCircle } from "react-icons/bs";
+
+interface VersionData {
+  buildDate: string;
+  buildSha: string;
+}
+
+const Footer: React.FC = () => {
```

Comment:
I'd keep the `React.FC` typing only in the places where we can pass children to the component

Reply:
there is a possibility, that the linter will complain about no type though :)

Reply:
It is unnecessary, but i would leave it for formalistic purposes